private_chaincode_apps/CMakeLists.txt

```diff
@@ -0,0 +1,37 @@
+# Copyright 2019 Intel Corporation
+# Copyright IBM Corp. All Rights Reserved.
+#
+# SPDX-License-Identifier: Apache-2.0
+
+cmake_minimum_required(VERSION 3.5.1)
+project(FPC_APPS)
```

Comment:
I have the feeling that FPC_APPS is not the right name here. this folder will just contain a chaincode, which may be part of a "bigger" blockchain application. I still like `examples`

Reply:
removed